app/views/taxa/index.html.erb

```diff
@@ -280,4 +280,5 @@
     </p>
     <% end %>
   </div>
+  <%= link_to "Taxonomy Details &raquo".html_safe, taxonomy_details_for_taxon_path(Taxon::LIFE) %>
```

Comment:
Needs to be internationalized if you want all public-facing stuff to be internationalized.